src/Symfony/ServiceMap.php

```diff
@@ -15,6 +15,6 @@ public function getServices(): array;
 
 	public function getService(string $id): ?ServiceDefinition;
 
-	public static function getServiceIdFromNode(Expr $node, Scope $scope): ?string;
+	public function getServiceIdFromNode(Expr $node, Scope $scope): ?string;
```

Comment:
Same as above for `ParameterMap` change, this might be considered a breaking change and thus require a major version bump?